examples/prod/README.md

```diff
@@ -0,0 +1 @@
+# Example deployment to a prod environment
```

Comment:
Instead of `prod` and `nonprod`, we can use `complete` as the dir name and then use `tfvars` file for each environment or even use a single tfvars file with a map of each environment and then a `for_each` on the module per environment